plugins/modules/source_control/gitlab/gitlab_branch.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Werner Dijkerman (ikben@werner-dijkerman.nl)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+module: gitlab_branch
+short_description: Create/delete branch
+description:
+  - Create/Delete branches
```

Comment:
Descriptions (except short_descriptions) should always contain full sentences (or multiple full sentences).
```suggestion
  - This module allows to create or delete branches.
```